plugins.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+declare -a PluginList=("query-monitor" "user-switching" "yoast-test-helper")
+
+# Install each plugin
+for plugin in "${PluginList[@]}"; do
+	echo Installing $plugin.
+	if [ -d "./plugins/$plugin/.git" ]; then
+		echo "Git clone found, not installing from WordPress.org."
+	else 
+		$(echo ./wp.sh plugin install $plugin --activate)
```

Comment:
Perhaps do a check here if `$plugin` is a git URL ( ends with `.git` or whatever ) and then run a git clone in the plugins directory followed by `$(echo ./wp.sh plugin activate $plugin)`.

Perhaps even running a `composer install` in that directory if a `composer.json` is found.